Source/UI/NewUI/PaymentForm/ViewModel/DefaultPaymentViewModel.swift

```diff
@@ -39,7 +42,13 @@ class DefaultPaymentViewModel: PaymentViewModel {
     }
   }
 
-  private func updateCardNumber() {
+  func getSupportedCheckoutSchema() -> [Card.Scheme] {
```

Comment:
suggestion : rename to `supportedCardSchemes`

Reply:
func should start with a verb.